src/patt-gui/pattern.hpp

```
@@ -19,7 +19,7 @@ typedef QVector < QVector < QPointF > > SPOTS; /*2d QPointF array - 1: layer, 2:
 #define LATTICE_GS 0
 #define LATTICE_SS 1
 typedef quint8 LATTICE;
-typedef quint8 DOMAIN;
+typedef quint8 DomainId;
```

Clang (14.0.6)

[warning] use 'using' instead of 'typedef'

(modernize-use-using)